djangochannelsrestframework/managers.py

```diff
@@ -0,0 +1,13 @@
+from typing import Any, Iterable, List, Optional
+from django.db.models import Manager
+from djangochannelsrestframework.signals import post_bulk_create
+
+
+class CustomManager(Manager):
+    def bulk_create(self, objs: Iterable[Any], **kwargs) -> List[Any]:
+        temp = super().bulk_create(objs, **kwargs)
+        print("bulk create", temp, objs)
+        for obj in temp:
+            print("loop", obj)
+            post_bulk_create.send(sender=obj.__class__, instance=obj, created=True)
```

Comment:
since your creating  custom signal maybe make that signal take all the objects in one go, much better than calling the signal 1000+ times

Reply:
I was investigating how to do this, maybe you are right, the signal should be call for all objects and be handled by the consumer using async.
BUT there is a problem with this aproach, we are limited by the backend database, that only in postgresql and mariadb10.5+, return the  `AutoField` values assigned.
I make more tests.

Reply:
it is ok if this bulk observation is only usable when being backed by postgresql or mariadb, i assume most deployments of django end up being backed by PG.

Reply:
Ok. `extras` module will be enough.